tableauserverclient/server/request_factory.py

```diff
@@ -958,6 +958,13 @@ def _generate_xml(
             views_element = ET.SubElement(workbook_element, "views")
             for view_name in workbook_item.hidden_views:
                 _add_hiddenview_element(views_element, view_name)
+
+        if workbook_item.thumbnails_user_id is not None:
+            workbook_element.attrib["thumbnailsUserId"] = workbook_item.thumbnails_user_id
+
+        if workbook_item.thumbnails_group_id is not None:
```

Comment:
only one of them should be set, so maybe make this an else if?

Reply:
sounds good, will change it to elif